index.js

```diff
@@ -58,6 +63,13 @@ extendConfig(function (config, userConfig) {
     }
 
     validate(conf, 'rename', 'function');
+
+    if (!config.format) {
+      conf.format = conf.pretty ? "minimal": "json";
+    }
```

Comment:
@ItsNickBarry let me know if you are good with this approach.
pretty internally just uses format and that way code is less messier in export_abi.js file

Reply:
Yes, this is the best way.  Similar to flat/rename above.